examples/java/src/test/java/dev/selenium/elements/LocatorsTest.java

```diff
@@ -1,7 +1,31 @@
 package dev.selenium.elements;
-
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.pagefactory.ByChained;
 import dev.selenium.BaseTest;
 
 public class LocatorsTest extends BaseTest {
 
+    WebDriver driver = new ChromeDriver();
+    // Navigate to Url
+    driver.get("https://www.selenium.dev/documentation/webdriver/elements/locators/");
+
+    // Find element using ByChained
+    // utilizes html described on page
+    By example = new ByChained(By.id("parentDiv"), By.id("phoneNumber"));
+    driver.findElements(example);
+
 }
+
+
+/* ByChained examples utilizes the following HTML located in locators.md
```

Comment:
Apologies if my earlier message was unclear...!

You can use any of the examples from [this page](https://www.selenium.dev/selenium/web/), and I'd appreciate having a working example. 

I think the [login example](https://www.selenium.dev/selenium/web/login.html) should be sufficient for demonstrating the concept.